### DISCUSSION TITLE
Categorization and tidying up

### DISCUSSION THREAD

**Discussion by Phredreeke** [opengaming/osgameclones#2357] [General]
There are two distinct concepts that are intermingled in the list, that of a clone and that of a port. A clone is in almost all cases playable without any original game, while ports in almost all cases requires original game assets to run.

I'm gonna use the game Blood as an example. NBlood, Raze and Transfusion are listed. Of those NBlood and Raze are ports, requiring the player to already own the game. Transfusion on the other hand is a partial recreation of Blood using the Quake engine, this would be categorised as a clone.

A fourth entry listed is XL engine. This IMO should never have been listed under any game other than Dark Forces and Daggerfall, as those are the only games that were playable in any publicly available build. This brings me to my second point, several entries have been superseded by other ports, and would be better either be culled or put on a category of its own. Examples would be ERampage (superseded by Rednukem and RedneckGDX), ZDoom (superseded by GZDoom), XL Engine (superseded by The Force Engine), having these listed among others are just gonna confuse the novice user.

**Comment by cxong** [comment]
Ok so there's several points you make, it would be better to split them. Also I'm moving this to discussion area for now, feel free to create issues if there's a specific thing to fix.
- clone / port: this is already covered by the "content" field. Usually games that require original assets have this as "commercial", and those that have their own assets are either "free" or "open". There is also "swappable" for games that support both original assets and free recreations, so it's not so cut and dry. I think others have argued in the past for adding a "source port" type, for games that used original source code, as opposed to a clean room implementation. Please find that discussion to find reasons why  we haven't added "source port".
- XL engine: please open an issue or PR; note that in future we may support listing these anyway but marking the originals under "unplayable" or some other status - see #596 
- Superseding games: we don't remove games that have been superseded; instead we can make a note on the old game or new game (info field) if this has happened. For example ZDoom mentions that GZDoom is one of its descendents

**Reply by Phredreeke** [reply]
Oops, I missed the discussions tab (none of the projects I've submitted bug reports to had it), you're right that this is the more appropriate place to raise this matter. I also failed to see the notes field.

I made a pull request for XL Engine but somehow I screwed up the formatting
